docs/user-guide/system-admin/configuration/installation.md

```diff
@@ -1,9 +1,10 @@
-Zanata can be installed by downloading a web archive (war) file, and configuring Jboss EAP or Wildfly according to this guide, or by downloading a handy installer.
+Zanata can be installed by downloading a web archive (war) file, and configuring Jboss EAP or Wildfly according to this guide, by downloading a handy installer, or using Docker.
```

Comment:
I'm not sure we should be pushing Docker yet, in view of limitations like https://zanata.atlassian.net/browse/ZNTA-1380. It's still pretty experimental and unsupported.